contracts/pcv/utils/FeiSkimmer.sol

```diff
@@ -53,4 +54,11 @@ contract FeiSkimmer is CoreRef {
         threshold = newThreshold;
         emit ThresholdUpdate(newThreshold);
     }
+
+    /// @notice Set the target to skim from. Only Governor or Admin
+    /// @param newSource the new source to skim from
+    function setSource(address newSource) external onlyGovernorOrAdmin {
```

Comment:
if you're going to use the onlyGovernorOrAdmin modifier, you have to set the admin of this contract in the constructor

Reply:
It's set by default in `CoreRef`, which this contract inherits from, to `GOVERNOR`. It's a bit redundant as the modifier allows GOVERNOR to interact with this method anyway but the contract admin could be changed in the future to something else. 

Should stay as is imo

Reply:
It's set by default in `CoreRef`, which this contract inherits from, to `GOVERNOR`. That's a bit redundant at the moment as the GOVERNOR can call this method anyway, but the contract admin could be changed in the future to something else. 

Should stay as is imo

Reply:
Have thought about this some more. In the recent governance upgrade, we changed the admin of the `feiSkimmer` contracts to the `PCV_MINOR_PARAM_ROLE`. That role is intended to only manage minor PCV related parameters and should not have the scope to change the source in my opinion.

Going to change `setSource` to be `onlyGovernor` and then `setContractAdmin` internally to `PCV_MINOR_PARAM_ROLE` to avoid having to do later in a DAO vote